test/CodeGen/RISCV/cbnez-cjalr.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: llc -mtriple=riscv32imac -verify-machineinstrs < %s | FileCheck %s
```

Comment:
We should start with a more minimal test case for these instructions (see test/CodeGen/RISCV/branch.ll).

Reply:
Add cbranch.ll to test c branch/jump generation.
Add cbranch-relaxation.ll to test c branch/jump relaxation.